qiskit/pulse/commands/acquire.py

```diff
@@ -106,8 +106,9 @@ def __init__(self,
             reg_slots = []
 
         # TODO: more precise time-slots
```

Comment:
Is this todo still required?

Reply:
This TODO is for the future when we have `acquisition_latency`.